heap.c

```diff
@@ -100,6 +110,20 @@ siftup(struct heap *h, size_t pos)
 }
 
 void
+heap_heapify(struct heap *h)
+{
+	for (ssize_t i=ptrvec_size(h->vec)/2-1; i>=0; i--) {
```

Comment:
`ptrvec_size()` returns a `size_t`, not a `ssize_t`, so use `size_t` here. Also please add some spaces. So:

```c
for (size_t i = ptrvec_size(h->vec) / 2 - 1; i >= 0; i--) {
```